index.rst

```diff
@@ -64,17 +54,19 @@ for discussion around a future HTTP client PSR.
 
 .. toctree::
     :hidden:
-    :maxdepth: 3
 
     PHP-HTTP <self>
 
 .. toctree::
    :hidden:
-
-   HTTPlug <httplug>
-
-.. toctree::
-   :hidden:
+   :caption: HTTPlug
```

Comment:
Well, these are captions (in subdued grey), not links. I changed this to show right away the Introduction, Usage and Tutorial links, which to me makes a lot of sense for doc readers.